drivers/clock_control/Kconfig.silabs

```diff
@@ -0,0 +1,11 @@
+# Silicon Labs Series 2+ CMU control driver
```

Comment:
This comment is redundant with help field. We can probably drop it.

Reply:
Removed.